app/controllers/v2/link_sets_controller.rb

```diff
@@ -12,6 +12,14 @@ def put_links
       render status: response.code, json: response
     end
 
+    def get_linked
+      render json: Queries::GetLinked.new(
+        content_id: content_id,
+        link_type: params.fetch(:link_type),
+        fields: params.fetch(:fields),
+      ).call
+    end
```

Comment:
If you don't provide a `link_type` or `fields` parameter, would you get a 500? Can we make this a 4xx response with a descriptive message?